app/views/hq/orphans/show.html.haml

```diff
@@ -0,0 +1,127 @@
+%div.action_items
+  %div.instance
+    = link_to 'Edit Orphan', edit_hq_orphan_path(@orphan.id),
+      class: 'btn btn-default', role: 'button'
+    = link_to 'Link to Orphan', '#',
+      class: 'btn btn-default', role: 'button'
+
+%dl.dl-horizontal
```

Comment:
Let's break this long display of attributes into thematic panels, the way it's currently done in the AA interface. Have a look at http://getbootstrap.com/components/#panels